### DISCUSSION TITLE
with dbeaver, how to show geom map of srid 26918?

### DISCUSSION THREAD

**Discussion by flyly0755** [dbeaver/dbeaver#19786] [Q&A]
dbeaver version: 23.0.0.202303040621
database: Postgres13.3
Use default LeafLet OpenStreetMap,
with srid 4326, dbeaver can show map of geom with detail info(such as roads, blocks and so on), for example geom showed as below:
POLYGON ((-71.1776585052917 42.3902909739571, -71.1776820268866 42.3903701743239, -71.1776063012595 42.3903825660754, -71.1775826583081 42.3903033653531, -71.1776585052917 42.3902909739571))

but with srid 26918, just show the geom self showed as below, without any backgroud and surrounding area, take try with other LeafLets, all the same.
MULTIPOLYGON (((579051.0303273386 4495284.647006003, 579062.1215490546 4495347.572287417, 579434.8797607257 4495581.098613336, 579437.0144684874 4495577.314539849, 579437.0233512535 4495577.298498718, 579437.040898537 4495577.26621309, 579437.0495630553 4495577.249968603, 579442.5441902657 4495566.751761724, 579449.1297679085 4495556.00091594, 579449.1391509742 4495555.985316758, 579449.1577043759 4495555.95390276, 579449.166874713 4495555.938087943, 579468.1317277302 4495522.6226703245, 579468.1383626282 4495522.610854133, 579468.1515136067 4495522.587108933, 579468.1580296955 4495522.575179936, 579482.6721488937 4495495.632976372, 579482.8031287086 4495494.720937029, 579482.4924496906 4495493.088592618, 579482.0507907424 4495492.368287573, 579090.1287079216 4495237.529030491, 579089.4253346118 4495237.501325827, 579088.2428480671 4495238.153693957, 579087.7637349295 4495238.833766716, 579086.4295151937 4495244.129347825, 579070.1348408061 4495272.638618323, 579070.119922648 4495272.665544377, 579070.0907040739 4495272.719964956, 579070.0764036581 4495272.747459479, 579064.3235466836 4495284.169352916, 579055.5587161769 4495278.773788461, 579054.9567888381 4495278.716155582, 579053.8745665858 4495279.142240854, 579053.3942717348 4495279.625958989, 579051.0303273386 4495284.647006003)))

Whether dbeaver not support show map of srid 26918 or need some extra settings?

**Comment by mkgrgis** [comment]
@flyly0755 , do you know any public TMS or WMTS map tile sources for SRID 26918 ? Does you ensure leaflet.js support this SRID?
